### PR TITLE
chore(css): Add CSS linting :)

### PR DESCRIPTION
**Pull request by redox**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "dev": "npm install && webpack-dev-server --config webpack.example.config.js --hot --inline --no-info",
     "doctoc": "doctoc --maxlevel 3 README.md",
     "gh-pages": "./scripts/gh-pages.sh",
-    "lint": "eslint . --quiet --no-color",
+    "lint": "eslint . --quiet --no-color && ./node_modules/stylelint/dist/cli.js './themes/*/*.css'",
```
Comment (vvo) [inline package.json]: No need for node_modules, everything in npm scripts will look for the binaries first in ./node_modules/.bin. SO you can just put "stylelint .."

File: package.json [hunk 2]
```diff
@@ -55,6 +55,7 @@
     "semver": "^5.0.3",
     "sinon": "sinonjs/sinon#a97374641599ed2f451e9a22f784060cf7a38b3e",
     "style-loader": "^0.12.4",
+    "stylelint": "^2.0.0",
```
Comment (vvo) [inline package.json]: need to npm-shrinkwrap: https://gist.github.com/vvo/d3c1c3d4358309ed2619

File: package.json (hunk 1 above)
Comment (redox) [inline package.json]: Oh cool :)

File: package.json (hunk 2 above)
Comment (redox) [inline package.json]: Arg indeed!

**Pull request merged**